src/hash/rescue/rpo/mod.rs

```diff
@@ -55,11 +55,11 @@ mod tests;
 pub struct Rpo256();
 
 impl Hasher for Rpo256 {
-    /// Rpo256 collision resistance is the same as the security level, that is 128-bits.
+    /// Rpo256 collision resistance is 127-bits.
     ///
     /// #### Collision resistance
     ///
-    /// However, our setup of the capacity registers might drop it to 126.
+    /// However, our setup of the capacity registers might drop it to 125.
     ///
     /// Related issue: [#69](https://github.com/0xPolygonMiden/crypto/issues/69)
     const COLLISION_RESISTANCE: u32 = 128;
```

Comment:
A few small comments:
1. I'd add a sentence or two explaining why collision resistance is 127 bits.
2. I probably wouldn't mention 125 bits as this is not a property of the hash function itself (we can mention it in other places).
3. Should we update `COLLISION_RESISTANCE` constant to $127$?
4. I'd probably remove the reference to the issue, as I think these updates address it.

Reply:
Done